code/game/objects/items/pinpointer.dm

```diff
@@ -38,6 +38,12 @@
 	toggle_on()
 	user.visible_message("<span class='notice'>[user] [active ? "" : "de"]activates [user.p_their()] pinpointer.</span>", "<span class='notice'>You [active ? "" : "de"]activate your pinpointer.</span>")
 
+/obj/item/pinpointer/examine(mob/user)
```

Comment:
Are you sure it's going to work? I already remember something similar runtiming in multitools.
There's also `on_examine()` function.

Reply:
yeah, tested it, lemme grab a screenshot real quick

Reply:
![image](https://user-images.githubusercontent.com/22431091/128645281-939dae32-5a39-4119-a984-3a338b7c915f.png)